Cargo.toml

```diff
@@ -17,7 +17,7 @@ This crate provides Grapheme Cluster, Word and Sentence boundaries
 according to Unicode Standard Annex #29 rules.
 """
 
-exclude = [ "target/*", "Cargo.lock", "scripts/tmp", "benches/texts/*", "*.txt", ]
+exclude = ["/.github", "/scripts", "/benches/texts"]
```

Comment:
I'd rather not exclude `scripts`, it's not used by the Rust package but people vendoring the crate sometimes care about having files like this. Can this be reverted to excluding `scripts/tmp`?

Reply:
Ahh ok, that's unfortunate, the main motivation for the change was to remove executable scripts, but I'll revert that part.

Reply:
What's the reason for removing executable scripts? In general I'm 100% sympathetic to the idea that one shouldn't ship unnecessary executables (and I'm aware of the recent kerfuffle in the broader community), but in this case (1) they're scripts, so they're easy to inspect and compare against the git history, and (2) they are not, to my knowledge, executed automatically under any circumstances.

Reply:
@Jake-Shadle if they have an executable bit set I do not mind removing that, but it's not a huge deal. They are scripts, not binaries.